conf/modules/nav_heli_spinup.xml

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="nav_heli_spinup" dir="nav">
+  <doc>
+    <description>This module controls the spinup of the main rotor from a Helicopter</description>
```

Comment:
Please try to remember that the first line of the description is taken as name (very brief descrption) in the generated http://docs.paparazziuav.org/latest/onboard_modules.html

Maybe we should really introduce a `<brief>Foo module</brief>` or so?